packages/components/test/SquiggleEditor.test.tsx

```diff
@@ -0,0 +1,34 @@
+import "@testing-library/jest-dom";
+
+import {
+  act,
+  getByText,
+  render,
+  screen,
+  waitFor,
+} from "@testing-library/react";
+
+import { SquiggleEditor } from "../src/index.js";
+
+test("Viewer renders", async () => {
+  act(() => render(<SquiggleEditor defaultCode="2 to 3" />));
+  await waitFor(() => screen.getByText(/simulation/));
+
+  expect(screen.getByText(/simulation/)).toHaveTextContent(
+    /simulation #(\d+) in (\d+)ms/
```

Comment:
minor, but I'd prefer it if `/simulation` and `/simulation #(\d+) in (\d+)ms/` were pulled out as constants.